.github/check_book_update.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Indicates whether the Programming Cryptol book PDF needs to be updated.
+#
```

Comment:
It might be helpful to include a usage guide that explains how one might run this script locally. (The `git diff` invocation one needs is somewhat non-obvious.)

Reply:
Good call, I added.